kyaml/fn/runtime/container/container.go

```diff
@@ -174,14 +177,18 @@ func (c *Filter) getCommand() (string, []string) {
 	if c.Network != "" {
 		network = c.Network
 	}
-
+	// run as nobody by default
+	user := c.User
+	if user == "" {
+		user = "nobody"
```

Comment:
```
const UserNobody = "nobody"
```
whereever the `User` type is defined.

Also wondering if this defaulting can happen when the Filter or whatever is created rather than just before the args are built.